### PR TITLE
Adrien be patch 4

### PR DESCRIPTION
**Pull request by adrien-be**

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -102,7 +94,7 @@ markdown_extensions:
 # Page tree
 nav:
   - Working Draft:
-      - 1. Introduction: index.md
+      - 1. Introduction: workingDraft/introduction.md
```
Comment (deepanshutr) [inline mkdocs.yml:97]: The landing page has to be named index in the top directory by mkdocs convention. Pl revert.

File: mkdocs.yml [hunk 2]
```diff
@@ -47,14 +47,6 @@ extra:
   social:
     - icon: fontawesome/brands/github
       link: https://github.com/interNFT
-    - icon: fontawesome/brands/twitter
```
Comment (deepanshutr) [inline mkdocs.yml:50]: pl revert

File: markdowns/workingGroup/gettingInvolved.md [hunk 3]
```diff
@@ -1,3 +1,10 @@
-# Getting Involved
+# Participate And Keep In Touch
```
Comment (deepanshutr) [inline markdowns/workingGroup/gettingInvolved.md:1]: Keeping in sync with w3c did website. also topic must be in sync with the file name.

File: mkdocs.yml (hunk 2 above)
Comment (adrien-be) [reply mkdocs.yml:50]: On it...

File: mkdocs.yml (hunk 1 above)
Comment (adrien-be) [reply mkdocs.yml:97]: On it...

File: markdowns/workingGroup/gettingInvolved.md (hunk 3 above)
Comment (adrien-be) [reply markdowns/workingGroup/gettingInvolved.md:1]: OK

**Review by deepanshutr — DISMISSED** [review]
Reviewed.

**Comment by deepanshutr** [comment]
conflict with master

**Review by deepanshutr — APPROVED** [review]
Reviewed.

**Pull request merged**